src/lib/components/Button/Button.scss

```diff
@@ -0,0 +1,160 @@
+  background: map.get($colors, "primary");
+  &:hover {
+    background: map.get($colors, "primary-dark");
+  }
+  &:active {
+    background: map.get($colors, "primary-darker");
+  }
+  &:disabled {
+    background: map.get($colors, "disabled-dark");
+  }
+}
+
+.usa-button--base,
+.usa-button.usa-button--base {
+  background: map.get($colors, "base-dark");
+  &:hover {
+    background: map.get($colors, "base-dark");
+  }
+  &:active {
+    background: map.get($colors, "base-darker");
+  }
+}
+
+.usa-button--inverse,
+.usa-button.usa-button--inverse {
+  color: map.get($colors, "primary");
+  background: #fff;
+  &:hover {
+    color: map.get($colors, "primary");
+    background: map.get($colors, "gray-05");
+  }
+  &:active {
+    color: map.get($colors, "primary");
+    background: map.get($colors, "primary-lightest");
+  }
+  &:disabled {
+    color: #fff;
+  }
+}
+
+.usa-button--text-only,
+.usa-button.usa-button--text-only {
+  color: map.get($colors, "primary");
+  background: none;
+  &:hover {
+    color: map.get($colors, "primary");
+    background: map.get($colors, "text-only-hover");
+  }
+  &:active {
+    color: map.get($colors, "primary");
+    background: map.get($colors, "text-only-active");
+  }
+  &:disabled {
+    background: none;
+    color: map.get($colors, "disabled");
+  }
+}
+
+.usa-button--outline,
+.usa-button.usa-button--outline {
+  background: none;
+  color: map.get($colors, "primary");
+  border: 2px solid map.get($colors, "primary");
+  box-shadow: none;
+  &:hover {
+    background: none;
+    color: map.get($colors, "primary-dark");
+    border: 2px solid map.get($colors, "primary-dark");
+    box-shadow: none;
+  }
+  &:active {
+    background: none;
+    color: map.get($colors, "base-darker");
+    border: 2px solid map.get($colors, "base-darker");
+    box-shadow: none;
+  }
+  &:disabled {
+    background: none;
+    color: map.get($colors, "disabled");
+    border: 2px solid map.get($colors, "disabled");
+  }
+}
+
+.usa-button--outline-inverse,
+.usa-button.usa-button--outline-inverse {
+  background: none;
+  color: map.get($colors, "outline-inverse");
+  border: 2px solid map.get($colors, "base-light");
+  &:hover {
+    background: none;
+    color: map.get($colors, "base-lightest");
+    border: 2px solid map.get($colors, "base-lightest");
+  }
+  &:active {
+    background: none;
+    color: #fff;
+    border: 2px solid #fff;
+  }
+  &:disabled {
+    background: none;
+    color: map.get($colors, "disabled-dark");
+    border: 2px solid map.get($colors, "disabled-dark");
+  }
+}
+
+.usa-button--big,
+.usa-button.usa-button--big {
+  background: map.get($colors, "primary");
+  color: #fff;
+  &:hover {
+    background: map.get($colors, "primary-dark");
+  }
+  &:active {
+    background: map.get($colors, "primary-darker");
+  }
+  &:disabled {
+    background: map.get($colors, "disabled");
+  }
+}
+
+.usa-button--big-inverse,
+.usa-button.usa-button--big-inverse {
+  @extend .usa-button--big;
```

Comment:
This doesn't seem to be working (at least not in the story):
<img width="1054" alt="Screenshot 2023-04-26 at 2 41 12 PM" src="https://user-images.githubusercontent.com/3094090/234685045-ce842116-4ea4-40af-82b4-163bbfdb4709.png">